connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -989,6 +1038,11 @@ export async function githubCodeSyncActivity({
         },
       });
 
+      await deleteDataSourceFolder({
+        dataSourceConfig,
+        folderId: getRepositoryInternalId(repoId),
+      });
+
```

Comment:
nit: I'd do the delete from core before the one in connectors, if something goes wrong you're better off with the entry in connectors and not the one in core than the other way around IMHO

Reply:
(feel free to not address ofc)